thislekan/oasis
# Redesign tables and redefine relationships

#### What does this PR do?

- This PR creates new tables and defines new relationships
- It refactors existing solutions to accomodate new relationships

#### Description of Task to be completed?

- change table relationship
- add lecturer, students, course history, session, result tables
- refactor user resolver, mutation, query files

#### How should this be manually tested?

- run `git fetch` and `git checkout ch-redesign-tables-relationship`.
- run `prisma reset` and click `y` on your keyboard after the prompt and then, press `Enter`
- run `prisma deploy`
- start the development server by running yarn start: dev.
- on your browser, visit `http://localhost:5001`
- all exisitng feature should be working perfectly

#### Any background context you want to provide?

- The above description assumes you have the project set up locally

#### What are the relevant pivotal tracker stories?

- N/A

#### Screenshots (if appropriate)

- N/A
